### PR TITLE
Rvc ci improvements

### PR DESCRIPTION
**Pull request by GregAC**

### CODE REVIEW

**Comment by GregAC** [comment]
This parallelises the riscv-compliance one (both by splitting it into two azure jobs for two suites and using -j4 on make to run multiple tests together on one agent).

It also incorporates the tweaks suggested on the previous PR plus has some additions to the README.md for the opentitan target in the riscv-compliance respository.

File: sw/vendor/patches/riscv_compliance/0004-Enable-parallel-runs.patch [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+diff --git a/Makefile b/Makefile
+index 25557c1..8f7b04a 100644
+--- a/Makefile
++++ b/Makefile
+@@ -50,6 +50,11 @@ else
+     ifeq ($(RISCV_TARGET),riscvOVPsim)
+         JOBS ?= -j8 --max-load=4
+     endif
++	  ifeq ($(RISCV_TARGET),opentitan)
++			ifeq ($(OT_TARGET), verilator)
```
Comment (imphil) [inline sw/vendor/patches/riscv_compliance/0004-Enable-parallel-runs.patch:20]: If you do a respin: 
```suggestion
+			ifeq ($(OT_TARGET),verilator)
```

File: sw/vendor/riscv_compliance/riscv-target/opentitan/README.md [hunk 2]
```diff
@@ -112,6 +113,27 @@ OK: 48/48
 There are several test suites that can be run `rv32i`, `rv32im`, `rv32imc` and `rv32Zicsr`.
 Change the `RISCV_ISA` argument passed to `make` to choose between them.
 
+## Changing targets
+When switching between targets (i.e. between FPGA and verilator) the `work` directory in the `riscv_compliance` tree must be remove to force a software rebuild.
```
Comment (imphil) [inline sw/vendor/riscv_compliance/riscv-target/opentitan/README.md:117]: READMEs generally wrap at 80 characters.

Comment (GregAC) [reply sw/vendor/riscv_compliance/riscv-target/opentitan/README.md:117]: Indeed, this one didn't though so matched the existing style (one line per sentence). I'll rewrap it all if I do end up making other changes.

**Pull request merged**